lib/view.rb

```diff
@@ -38,19 +38,16 @@ def create output_dir
 
     create_inqlude_all(output_dir)
 
-
     @root = ""
 
-    render_template "index", output_dir
-    render_template "development", output_dir
-    render_template "unreleased", output_dir
-    render_template "commercial", output_dir
-    render_template "all", output_dir
-    render_template "about", output_dir
-    render_template "get", output_dir
-    render_template "contribute", output_dir
-    render_template "search", output_dir
-    
+    Dir.glob("#{view_dir}*.html.haml") do |file|
+      fname = File.basename file
+      fname.slice! ".html.haml"
```

Comment:
Changing the content in place is not necessary. You could just do something like `filename = File.basename(file).slice`. Then the meaning of the variable doesn't change within the method.